app/templates/custom-elements/remote-screen.html

```diff
@@ -108,8 +118,28 @@
           screenImg.addEventListener("contextmenu", (evt) => {
             evt.preventDefault();
           });
-
-          window.addEventListener("resize", this.onWindowResize);
+          let isTouchScreen = false;
+          this.shadowRoot.addEventListener("touchend", () => {
+            isTouchScreen = true;
+          });
+          this.shadowRoot.addEventListener("click", () => {
+            if (isTouchScreen) {
+              this.shadowRoot.getElementById("mobile-keyboard-input").focus();
+            }
+          });
+          this.shadowRoot
+            .getElementById("mobile-keyboard-input")
+            .addEventListener("keyup", (evt) => {
```

Comment:
Can we add a comment explaining this? I'm having trouble understanding the logic here. Also, how do we detect backspace? `innerHTML` can't be a backspace, can it?

Reply:
Anything not 229 like backspace gets sent the original way.